.sequelizerc

```diff
@@ -4,5 +4,6 @@ module.exports={
     "config": path.resolve('./src/sequelize/config', 'config.js'),
     "models-path": path.resolve('./src/sequelize/models'),
     "seeders-path": path.resolve('./src/sequelize/seeders'),
+    "migrations-source-path": path.resolve('./src/sequelize/migrations'),
```

Comment:
what is this for?

Reply:
Its needed by sequelize cli for you to use the seed:generate command